tools/pipelines/templates/include-test-task.yml

```diff
@@ -10,39 +10,17 @@ parameters:
 - name: buildDirectory
   type: string
 
-- name: testCoverage
-  type: boolean
-  default: false
```

Comment:
I think most pipelines that are not `build - client` end up passing true to this, so if we remove it we would be changing the way tests are run in all of those, no? Let's compare runs of another build pipeline pre/post this change and see that the way tests are run there hasn't changed.